packages/themes/src/g80.js

```diff
@@ -0,0 +1,316 @@
+export const layerSelectedHover = adjustLightness(layerSelected, -6);
+export const layerSelectedInverse = gray10;
+export const borderSubtleSelected = gray50;
+
+export const layerDisabled = gray70;
+export const fieldDisabled = gray70;
+export const borderDisabled = gray70;
+
+export const textDisabled = gray50;
+export const buttonDisabled = gray50;
+export const iconDisabled = gray50;
+
+export const textOnColorDisabled = gray30;
+export const iconOnColorDisabled = gray30;
+export const layerSelectedDisabled = gray30;
+
+export const skeletonBackground = adjustLightness(background, +6);
+export const skeletonElement = gray60;
+
+//////////////////////////////////////
+// Old tokens needed to pass tests  //
+//////////////////////////////////////
+export const interactive01 = backgroundBrand;
+export const interactive02 = buttonSecondary;
+export const interactive03 = buttonTertiary;
+export const interactive04 = interactive;
+
+export const uiBackground = background;
+
+export const ui01 = layer;
+export const ui02 = gray70;
+export const ui03 = layerAccent;
+export const ui04 = borderStrong;
+export const ui05 = borderInverse;
+
+export const text01 = textPrimary;
+export const text02 = textSecondary;
+export const text03 = textPlaceholder;
+export const text04 = textOnColor;
+export const text05 = textHelper;
+
+export const icon01 = iconPrimary;
+export const icon02 = iconSecondary;
+export const icon03 = iconOnColor;
+
+export const link01 = linkPrimary;
+export const link02 = linkSecondary;
+
+export const inverseLink = linkInverse;
+
+export const field01 = field;
+export const field02 = gray70;
+
+export const inverse01 = textInverse;
+export const inverse02 = backgroundInverse;
+
+export const support01 = supportError;
+export const support02 = supportSuccess;
+export const support03 = supportWarning;
+export const support04 = supportInfo;
+
+export const inverseSupport01 = supportErrorInverse;
+export const inverseSupport02 = supportSuccessInverse;
+export const inverseSupport03 = supportWarningInverse;
+export const inverseSupport04 = supportInfoInverse;
+
+export const overlay01 = overlay;
+
+export const danger01 = buttonDangerPrimary;
+export const danger02 = buttonDangerSecondary;
+
+// Interaction states
+export const inverseFocusUi = focusInverse;
+
+export const hoverPrimary = buttonPrimaryHover;
+export const activePrimary = buttonPrimaryActive;
+
+export const hoverPrimaryText = linkPrimaryHover;
+
+export const hoverSecondary = '#606060';
+export const activeSecondary = gray80;
+
+export const hoverTertiary = gray10;
+export const activeTertiary = gray30;
+
+export const hoverUI = backgroundHover;
+export const hoverLightUI = '#656565';
+export const activeUI = backgroundActive;
+export const activeLightUI = gray50;
+export const selectedUI = backgroundSelected;
+export const selectedLightUI = gray60;
+export const inverseHoverUI = backgroundInverseHover;
+
+export const hoverSelectedUI = backgroundSelectedHover;
```

Comment:
Change this to `layerSelectedHover`. In v10 `hoverSelectedUI` works for both background and ui-01, it need ti work for both so use the lighter option.